exporters/mongodb3/mongodb3.json.tmpl

```diff
@@ -51,9 +51,7 @@
 
         "--collector.collstats-limit=0",
 
-        {{ if .config.exporter_port}}
         "--web.listen-address",
-        ":{{.config.exporter_port}}",
-        {{end}}
+        ":{{.exporter_port}}",
```

Comment:
would it worth adding a comment line saying that it is expected:
 - not being config.exporter_port
 - expected to be populated by the configurator and therefore there is no if

Reply:
Updated here: [846d204](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/162/commits/846d204934f4268dbdeca775844aa26b10104c09)